packages/venia-concept/src/components/PurchaseDetailsPage/OrderItemsList/OrderItemsList.js

```diff
@@ -1,26 +1,62 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { List } from '@magento/peregrine';
+
 import OrderItem from '../OrderItem';
+import classify from 'src/classify';
+import defaultClasses from './orderItemsList.css';
 
-export default class OrderItemsList extends Component {
+class OrderItemsList extends Component {
     static propTypes = {
-        items: PropTypes.array,
-        title: PropTypes.node
+        classes: PropTypes.shape({
+            body: PropTypes.string,
+            itemsContainer: PropTypes.string
+        }),
+        title: PropTypes.string,
+        items: PropTypes.arrayOf(
+            PropTypes.shape({
+                id: PropTypes.number,
+                title: PropTypes.string,
+                size: PropTypes.string,
+                color: PropTypes.string,
+                qty: PropTypes.number,
+                titleImageSrc: PropTypes.string,
+                price: PropTypes.number
+            })
+        ),
+        onBuyAgain: PropTypes.func,
+        onShare: PropTypes.func
     };
 
     static defaultProps = {
-        items: []
+        title: 'Other Items in this Order'
```

Comment:
Remove this after rebasing on the latest version of feature/purchase-details-page